Makefile

```diff
@@ -29,13 +29,13 @@ pre-commit-run:
 	@pre-commit run --all-files
 
 unit-tests:
-	@uv run pytest -svv tests/unit/*
+	@uv run pytest -svv tests/unit/
 
 functional-tests:
-	@uv run pytest -svv tests/functional/*
+	@uv run pytest -svv tests/functional/
 
 e2e-tests:
-	@uv run pytest -svv tests/e2e/*
+	@uv run pytest -svv tests/e2e/
 
 tests: unit-tests functional-tests
```

Comment:
Since you are already updating the macros: can you add `e2e-tests` here as well?